chef/cookbooks/crowbar/recipes/stop-services-before-upgrade.rb

```diff
@@ -40,7 +40,9 @@
   code <<-EOF
     for resource in g-rabbitmq rabbitmq fs-rabbitmq ms-drbd-rabbitmq drbd-rabbitmq;
     do
-      crm --force --wait resource stop $resource
+      if crm configure show | grep -q $resource; then
```

Comment:
Please do `if crm configure show $resource` instead. Your grep is matching too many things ("grep rabbitmq" for instance)